### PR TITLE
Chef 13 compatibility fixes

### PR DESCRIPTION
**Pull request by jcorme**

### CODE REVIEW

**Comment by jcorme** [comment]
[Chef 13: ganeti](https://trello.com/c/InvZZyL5/917-chef-13-ganeti)

File: .kitchen.yml [hunk 1]
```diff
@@ -29,9 +29,6 @@ transport:
 platforms:
   - name: centos-6.9
   - name: centos-7.3
```
Comment (ramereth) [inline .kitchen.yml]: Please update these to 6.10 and 7.5

**Comment by jcorme** [comment]
!bump major *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v2.0.0.  Have a nice day!

**Pull request merged**